cli/cobradaptor/adaptor.go

```diff
@@ -0,0 +1,84 @@
+package cobradaptor
+
+import (
+	"github.com/docker/docker/api/client"
+	"github.com/docker/docker/api/client/volume"
+	"github.com/docker/docker/cli"
+	cliflags "github.com/docker/docker/cli/flags"
+	"github.com/docker/docker/pkg/term"
+	"github.com/spf13/cobra"
+)
+
+// CobraAdaptor is an adaptor for supporting spf13/cobra commands in the
+// docker/cli framework
+type CobraAdaptor struct {
```

Comment:
It feels a little weird to me that the struct is called `CobraAdaptor` but the package name is `cobradaptor` (one fewer `a`). I don't have anything against the portmanteau but I'd rather not have two confusingly similar names.